### PR TITLE
Automation ansible

### PR DESCRIPTION
**Pull request by son-vyas**
Please do review and suggest changes if any.

### CODE REVIEW

File: chapter_3/automation_using_ansible.md [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
-  - Playbooks
-  - Roles
-  - Inventory
-  - Job and Job Template
-- Ruby and Ansible
+### ManageIQ and Ansible
+
+Before ease of Ansible reached out to us, one was expected to be quite comfortable with Ruby scripting language and [Automate object model](https://pemcg.gitbooks.io/mastering-automation-in-cloudforms-4-2-and-manage/content/introduction_to_the_automate_datastore/chapter.html) to develop his own automation scripts. But, Now ManageIQ do provide varios provisioning and service management processes. Thus, after Fine release, we can use Ansible playbooks for automation.
+
+#### Ansible Terms
```
Comment (akshaygin) [inline chapter_3/automation_using_ansible.md]: This one should have h3(###).
And all the terms under this, should have h4(####)

File: chapter_3/automation_using_ansible.md [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
+
+##### Modules
+Ansible modules are reusable scripts that is used by Ansible or Ansible-playbooks. They do return information to ansible by printing a JSON string to stdout before exiting.
```
Comment (akshaygin) [inline chapter_3/automation_using_ansible.md]: Also for the cleanliness, keep one blank line after each term.

File: chapter_3/automation_using_ansible.md [hunk 3]
```diff
@@ -5,3 +5,24 @@ Ansible is an opensource tool which serves for configuration management, orchest
 ### ManageIQ and Ansible
 
 Before ease of Ansible reached out to us, one was expected to be quite comfortable with Ruby scripting language and [Automate object model](https://pemcg.gitbooks.io/mastering-automation-in-cloudforms-4-2-and-manage/content/introduction_to_the_automate_datastore/chapter.html) to develop his own automation scripts. But, Now ManageIQ do provide varios provisioning and service management processes. Thus, after Fine release, we can use Ansible playbooks for automation.
```
Comment (KhushbuB) [inline chapter_3/automation_using_ansible.md]: Please correct the misspelled word 'various'.

**Comment by akshaygin** [comment]
Well written. Good work @son-vyas

File: chapter_3/automation_using_ansible.md [hunk 4]
```diff
@@ -1,11 +1,40 @@
...
-- Ansible Terms
-  - Modules
-  - Playbooks
-  - Roles
-  - Inventory
-  - Job and Job Template
-- Ruby and Ansible
+## Introduction to Ansible
+
+Ansible is an opensource tool which serves for configuration management, orchestration, provisioning and deployments. It is developed using python and runs on Windows, Mac, and UNIX-like systems. The Ansible itself is agentless i.e you don't need to install any client side software.It uses SSH connections. Hence,if you have well-oiled SSH setup, then you are roll up to use ansible in your environment. This also means that you can install it on only one machine and control your infrastructure from that machine itself.
```
Comment (psachin) [inline chapter_3/automation_using_ansible.md]: Use "Open source" instead of "opensource"

**Review by psachin — CHANGES_REQUESTED** [review]

File: chapter_3/automation_using_ansible.md (hunk 4 above)
Comment (son-vyas) [reply chapter_3/automation_using_ansible.md]: Done.

**Comment by akshaygin** [comment]
@KhushbuB Any changes in this?

**Comment by KhushbuB** [comment]
LGTM :)

**Pull request merged**